packages/create-svelte/shared/+vitest/src/lib/sum.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Add two numbers
+ * @param {number} a
+ * @param {number} b
+ */
+export function sum(a: number, b: number): number {
```

Comment:
it's weird that this isn't used anywhere except in the tests

One idea might be to convert the `<a href="https://kit.svelte.dev">kit.svelte.dev</a>` used in `+page.svelte` to a function like `{@html linkify('kit.svelte.dev')}`:
```
export function linkify(domain: string) {
  return `<a href="https://${domain}">${domain}</a>`;
}
```